A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -3029,7 +3029,8 @@ class FMCMainDisplay extends BaseAirliners {
      * @returns {number} dynamic delay in ms between 2000ms and 4000ms
      */
     getDelayFuelPred() {
-        return 1714 * this.flightPlanManager.getWaypointsCount() + this.flightPlanManager.getDestination().cumulativeDistanceInFP;
+        const tripDist = this.flightPlanManager.getDestination().cumulativeDistanceInFP;
+        return 225 * this.flightPlanManager.getWaypointsCount() + (tripDist > 0 ? tripDist / 2 : 0);
```

Comment:
What is this ternary statement for? Can the cumulative distance get negative?

Reply:
I thought there'd be an exception if I tried to divide by 0 but I tried it out in jsfiddle and it's ok

Reply:
You're not dividing by zero, you're dividing 0 by two ;)